lib/node_modules/@stdlib/iter/cusome-by/README.MD

```diff
@@ -0,0 +1,238 @@
+
+<section class="notes">
+
+## Notes
+
+-   If an environment supports `Symbol.iterator`, the returned iterator is iterable.
+-   The returned iterator continues iterating until it reaches the end of the input iterator, even after the condition is met.
+
+</section>
+
+<!-- /.notes -->
+
+<!-- Package usage examples. -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var randu = require( '@stdlib/random/iter/randu' );
+var iterCuSomeBy = require( '@stdlib/iter/cusome-by' );
+
+function threshold( r ) {
+    return ( r >= 0.95 );
+}
+
+// Create an iterator which generates uniformly distributed pseudorandom numbers:
+var opts = {
+    'iter': 100
+};
+var riter = randu( opts );
+
+// Create an iterator which iteratively checks for values above a threshold:
+var it = iterCuSomeBy( riter, 5, threshold );
+
+// Perform manual iteration...
+var v;
+while ( true ) {
+    v = it.next();
+    if ( v.done ) {
+        break;
+    }
+    console.log( v.value );
+}
+```
+
+</section>
+
+<!-- /.examples -->
+
+<!-- Section to include cited references. If references are included, add a horizontal rule *before* the section. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="references">
+
+</section>
+
+<!-- /.references -->
+
+<!-- Section for related `stdlib` packages. Do not manually edit this section, as it is automatically populated. -->
+
+<section class="related">
+
+* * *
+
+## See Also
+
+-   <span class="package-name">[`@stdlib/iter/cuany-by`][@stdlib/iter/cuany-by]</span><span class="delimiter">: </span><span class="description">create an iterator which iteratively applies a test function to iterated values, returning `true` if a test is passed and `false` otherwise.</span>
+-   <span class="package-name">[`@stdlib/iter/cuevery-by`][@stdlib/iter/cuevery-by]</span><span class="delimiter">: </span><span class="description">create an iterator which iteratively applies a test function to iterated values, returning `false` if a test fails and `true` otherwise.</span>
+-   <span class="package-name">[`@stdlib/iter/for-each`][@stdlib/iter/for-each]</span><span class="delimiter">: </span><span class="description">create an iterator which invokes a function for each iterated value before returning the iterated value.</span>
+-   <span class="package-name">[`@stdlib/iter/cunone-by`][@stdlib/iter/cunone-by]</span><span class="delimiter">: </span><span class="description">create an iterator which iteratively applies a test function to iterated values, returning `true` if all values fail and `false` otherwise.</span>
+-   <span class="package-name">[`@stdlib/iter/cusome`][@stdlib/iter/cusome]</span><span class="delimiter">: </span><span class="description">create an iterator which iteratively checks whether at least `n` iterated values are truthy.</span>
```

Comment:
Do not manually edit this section. See the comment at L197. This section should be empty.